pkg/distributor/distributor.go

```diff
@@ -305,6 +305,8 @@ func (d *Distributor) running(ctx context.Context) error {
 }
 
 func (d *Distributor) cleanupMetricsForUser(userID string) {
```

Comment:
It doesn't cleanup only metrics now. We could rename the function for more clarity.